### PR TITLE
Added support of -w flag in 'scw restart' (Fix #185)

### LINKED ISSUE #185: Add support of -w flag in 'scw restart'

### PR DESCRIPTION
**Pull request by aimxhaisse**

### CODE REVIEW

File: pkg/commands/restart.go [hunk 1]
```diff
@@ -6,32 +6,80 @@ package commands
...
+				}
+				res = ""
+			} else {
+				if wait {
+					// FIXME: handle gateway
+					api.WaitForServerReady(ctx.API, server, "")
+				}
+			}
+			cr <- res
+			wg.Done()
```
Comment (QuentinPerez) [inline pkg/commands/restart.go]: `defer wg.Done()` at the line: 29

**Comment by moul** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1130,6 +1132,7 @@ $ scw inspect myserver | jq '.[0].public_ip.address'
 ### master (unreleased)
 
 * Support of `scw _userdata name VAR=@/path/to/file` ([#183](https://github.com/scaleway/scaleway-cli/issues/183))
+* Support of `scw restart -w` ([#185](https://github.com/scaleway/scaleway-cli/issues/185))
```
Comment (moul) [inline README.md]: `+ restarting multiple servers in parallel`

File: pkg/commands/restart.go (hunk 1 above)
Comment (aimxhaisse) [inline pkg/commands/restart.go]: done

File: README.md (hunk 2 above)
Comment (aimxhaisse) [inline README.md]: done

**Comment by QuentinPerez** [comment]
LGTM

**Pull request merged**